### PR TITLE
MES-2771: Populate email template A mod1

### PR DESCRIPTION
**Pull request by szabikr**
- Configuring new providers
- Getting the template ID
- Extracting custom properties from Test Report and adding it to the e-mail/post (Speed Checks)
- Extracting SingleFaultCompetency Faults and Speed Check Faults
- Updating labels

### CODE REVIEW

File: src/functions/sendCandidateResults/application/service/categories/AM1/custom-property-provider-cat-a-mod1.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      emergencyStopAttempts.emergencySecondAttempt = emergencyStopData.secondAttempt!.toString();
+    }
+  }
+
+  return emergencyStopAttempts;
+};
+
+const getAvoidanceAttempts = (
+  avoidanceData: CatAMod1TestData['avoidance'],
+): Pick<
```
Comment (TomBillingtonUK) [inline src/functions/sendCandidateResults/application/service/categories/AM1/custom-property-provider-cat-a-mod1.ts:68]: not seen this before - nice :)

File: src/functions/sendCandidateResults/application/service/personalisation-provider.ts [hunk 2]
```diff
@@ -32,11 +33,13 @@ export class PersonalisationProvider implements IPersonalisationProvider {
 
   constructor(
     @inject(TYPES.IFaultProvider) private faultProvider: IFaultProvider,
+    @inject(TYPES.ICustomPropertyProvider) private customPropertyProvider: ICustomPropertyProvider,
   ) { }
 
   public getEmailPersonalisation(testresult: TestResultSchemasUnion): EmailPersonalisation {
     const sharedValues: Personalisation = this.getSharedPersonalisationValues(testresult);
 
+    console.log('##### Shared Values:', JSON.stringify(sharedValues));
```
Comment (TomBillingtonUK) [inline src/functions/sendCandidateResults/application/service/personalisation-provider.ts]: console log

**Review by TomBillingtonUK — APPROVED** [review]
Nice Work

File: src/functions/sendCandidateResults/application/service/categories/AM1/custom-property-provider-cat-a-mod1.ts (hunk 1 above)
Comment (szabikr) [reply src/functions/sendCandidateResults/application/service/categories/AM1/custom-property-provider-cat-a-mod1.ts:68]: Yeah, this is @AndyPerryKainos . Good work!

File: src/functions/sendCandidateResults/application/service/personalisation-provider.ts (hunk 2 above)
Comment (szabikr) [reply src/functions/sendCandidateResults/application/service/personalisation-provider.ts]: will remove

**Review by appleabs — APPROVED** [review]

**Pull request merged**